presto-spi/src/main/java/com/facebook/presto/spi/type/Chars.java

```diff
@@ -27,6 +30,50 @@ public static boolean isCharType(Type type)
         return type instanceof CharType;
     }
 
+    public static Slice padSpacesAndTruncateToLength(Slice slice, Type type)
+    {
+        requireNonNull(type, "type is null");
+        if (!isCharType(type)) {
+            throw new IllegalArgumentException("type must be the instance of CharType");
+        }
+        return padSpacesAndTruncateToLength(slice, CharType.class.cast(type));
+    }
+
+    public static Slice padSpacesAndTruncateToLength(Slice slice, CharType charType)
+    {
+        requireNonNull(charType, "charType is null");
+        return padSpacesAndTruncateToLength(slice, charType.getLength());
+    }
+
+    public static Slice padSpacesAndTruncateToLength(Slice slice, int maxLength)
```

Comment:
Seems like the parameter should be named `length` not `maxLength`, since the resulting string will always be `maxLength` never less